vendor/github.com/GoogleCloudPlatform/gke-networking-api/apis/network/v1/annotations.go

```diff
@@ -212,6 +246,13 @@ func ParseNICInfoAnnotation(annotation string) (NICInfoAnnotation, error) {
 	return *ret, err
 }
 
+// ParseInterfaceStatusAnnotation parses the given annotation to InterfaceStatusAnnotation
+func ParseInterfaceStatusAnnotation(annotation string) (InterfaceStatusAnnotation, error) {
+	ret := &InterfaceStatusAnnotation{}
+	err := json.Unmarshal([]byte(annotation), ret)
```

Comment:
suggest, to avoid partially converted results...

if err := json.Unmarshal(); err != nil { return InterfaceStatusAnnotation{}, err }
return ret, nil

Reply:
yes, this has to be fixed in https://github.com/GoogleCloudPlatform/gke-networking-api